code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -309,6 +314,44 @@
 	M.adjustBrainLoss(5, 150)
 	holder.remove_reagent(id, 1)
 
+/datum/reagent/fuel/holyoil		//Its oil
+	name = "Zelus Oil"
+	id = "holyoil"
+	description = "Oil blessed by a grater being."
+	taste_description = "oils"
+
+/datum/reagent/fuel/holyoil/on_mob_life(mob/living/carbon/M)
+	if(is_servant_of_ratvar(M))
+		M.drowsyness = max(M.drowsyness-5, 0)
+		M.AdjustUnconscious(-60, 0)
+		M.AdjustStun(-30, 0)
+		M.AdjustKnockdown(-70, 0)
+		M.adjustStaminaLoss(-15, 0)
+		M.adjustToxLoss(-5, 0)
+		M.adjustOxyLoss(-3, 0)
+		M.adjustBruteLoss(-3, 0)
+		M.adjustFireLoss(-5, 0)
+	if(iscultist(M))// We REALLY hate blood cult
+		M.AdjustUnconscious(1, 0)
+		M.AdjustStun(10, 0)
+		M.AdjustKnockdown(20, 0)
+		M.adjustStaminaLoss(15, 0)
+	else	//Were not mean
+		M.adjustToxLoss(3, 0)
+		M.adjustOxyLoss(2, 0)
+		M.adjustStaminaLoss(10, 0)
+	holder.remove_reagent(id, 1)
+	return TRUE
+
+//We only get 30u to start with...
+
+/datum/reagent/fuel/holyoil/reaction_obj(obj/O, reac_volume)
+	if(istype(O, /obj/item/stack/sheet/metal))
```

Comment:
add in the parent call `. = ..()` beforehand, even if it doesn't have much of an effect anyway currently.